items/MIDAS_STAFF.json

```diff
@@ -26,15 +25,16 @@
     "§7is §316000 §7if the bid was",
     "§7§6100,000,000 Coins §7or higher!",
     "",
+    "§7Price paid: §60 Coins",
+    "§7Base Ability Damage Bonus: §30",
+    "",
     "§7§8This item can be reforged!",
-    "§7§4❣ §cRequires §aCatacombs Level 25",
-    "§a§cwhile in Catacombs",
```

Comment:
Can you add the reqs back, other than that it looks good

Reply:
But you don't need cata lvl unless it's stared?

Reply:
Oh okay